giskard/testing/tests/calibration.py

```diff
@@ -42,7 +41,6 @@ def test_overconfidence_rate(
     slicing_function: Optional[SlicingFunction] = None,
     threshold: Optional[float] = 0.10,
     p_threshold: Optional[float] = None,
-    debug: bool = False,
```

Comment:
Removing these would break backward compatibility.
The old test suites cannot be executed due to:
```
Traceback (most recent call last):
  File "/Users/inoki/Builds/giskard/giskard/core/suite.py", line 380, in run
    result = test_partial.giskard_test.get_builder()(**test_params).execute()
  File "/Users/inoki/Builds/giskard/giskard/ml_worker/testing/registry/giskard_test.py", line 146, in execute
    return self.test_fn(**self.params)
  File "/Users/inoki/Builds/giskard/giskard/testing/tests/metamorphic.py", line 268, in test_metamorphic_invariance
    output_sensitivity=output_sensitivity,
TypeError: _test_metamorphic() got an unexpected keyword argument 'debug'
```

Reply:
Resolved by supporting legacy debug mode